ecchronos-binary/src/resources/jvm.options

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright 2020 Telefonaktiebolaget LM Ericsson
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#################################################################
+#                             jvm.options                       #
+#                                                               #
+# - all flags defined here will be used by to startup the JVM   #
+# - one flag should be specified per line                       #
+# - lines that do not start with '-' will be ignored            #
+# - only static flags are accepted (no variables or parameters) #
+#################################################################
+
+########################
+# GENERAL JVM SETTINGS #
+########################
+
+# Make sure all memory is faulted and zeroed on startup.
+# This helps prevent soft faults in containers and makes
+# transparent hugepage allocation more effective.
+#-XX:+AlwaysPreTouch
+
+#################
+# HEAP SETTINGS #
```

Comment:
Another useful flag could be:

`-XX:ParallelGCThreads=1` to avoid situation like https://issues.apache.org/jira/browse/CASSANDRA-14475 / https://bugs.java.com/bugdatabase/view_bug.do?bug_id=6858051

Reply:
Done